### DISCUSSION TITLE
Allow "Hide Top Toolbar" to be enabled separately

### DISCUSSION THREAD

**Discussion by ghost** [zen-browser/desktop#227] [Ideas]
![Screenshot from 2024-08-11 16-45-25](https://github.com/user-attachments/assets/85a7e189-658b-4031-88b4-09dbe81d690a)
Would love to be able to hide the top toolbar without having full compact mode enabled. I like being able to see my tabs, but want to hide the address bar.

**Comment by mr-cheffy** [comment]
Im already working on it as many people also requested it, thanks!

> im gonna close this to do some cleaning up
